src/main.rs

```diff
@@ -188,8 +188,8 @@ fn install_hook(color_scheme: ColorScheme) -> Result<(), Box<dyn Error>> {
         Err(e) if e.kind() == std::io::ErrorKind::AlreadyExists => {
             eprintln!(
                 "Failed to create `{HOOK_PATH}` as it already exists. \
-                Please either or remove it and re-run `gimoji -i` or \
-                add the following commandline to it:\n{content}",
+                Please either remove it and re-run `gimoji -i`, or \
+                add the following command-line to it:\n{content}",
```

Comment:
that's german.

command line is english

Reply:
https://en.wikipedia.org/wiki/Command-line_interface

Reply:
that page on the `en` subdomain, has "command-line" all over it.

Reply:
it's "a command line", but "a command-line interface"